scripts/deploy.sh

```
@@ -83,18 +84,81 @@ if [ "$SUCCESS" != "true" ]; then
 fi
 
-# Nginx 전환 (sites-enabled/app.conf 심볼릭 링크 교체)
-log "Nginx 설정을 $IDLE_COLOR 기준으로 전환"
+# Nginx 전환 (upstream.conf 스위치)
+log "Nginx upstream을 $IDLE_COLOR 기준으로 전환 (port=$IDLE_PORT)"
 
-if [ "$IDLE_COLOR" = "blue" ]; then
-  sudo ln -sf /etc/nginx/sites-available/app-blue.conf /etc/nginx/sites-enabled/app.conf
+# 전환 직전 최종 확인(짧은 재시도: 3회, 1초 간격)
+PRECHECK_OK="false"
+for j in {1..3}; do
```

Shellcheck (0.11.0)

[warning] j appears unused. Verify use (or export if used externally).

(SC2034)